neuro_san/message_processing/basic_message_processor.py

```diff
@@ -65,3 +67,27 @@ def get_chat_context(self) -> Dict[str, Any]:
                 Empty dictionaries or None values simply start a new conversation.
         """
         return self.chat_context.get_chat_context()
+
+    def get_structure(self) -> Dict[str, Any]:
+        """
+        :return: Any dictionary structure that was contained within the final answer
+                 from the agent session interaction, if such a specific breakout was desired.
+        """
+        return self.answer.get_structure()
+
+    def get_compiled_answer(self) -> str:
+        """
+        :return: All components of the "answer" except for sly_data compiled into
+                 a single string.  This includes the results of these methods in order:
+                    get_answer()
+                    get_structure()
+        """
```

Comment:
This is a bit of logic I found myself repeating for the simple clients, so it's consolidated here for standardized output.